llvm/lib/Target/AIE/AIE2Subtarget.cpp

```diff
@@ -80,6 +80,7 @@ InstructionSelector *AIE2Subtarget::getInstructionSelector() const {
 std::bitset<8>
 AIE2Subtarget::getMemoryBanksFromAddressSpace(unsigned AddrSpace) const {
   std::bitset<8> MemoryBanks;
+  MemoryBanks.reset();
```

Comment:
Please move to the previous commit. But that means that by default a `std::bitset` is uninitialized?

Reply:
Have removed this , by default `bitset` is initialized to 0.